src/Models/Keyword.php

```diff
@@ -23,6 +24,11 @@ protected static function boot()
         static::saving(function ($keyword) {
             $keyword->phrase = strtolower($keyword->phrase);
         });
+
```

Comment:
Are we sure we want to add this as a global scope? It seems like this should just be a customer query builder or local scope since the user is going to want to see their inactive keywords somewhere

Reply:
I was trying to do it like the locations package. But I think you are allowed to do something like static:: withoutGlobalScope('active')->get() right?